src/Goerli-DssSpell.sol

```diff
@@ -40,59 +45,28 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
+    //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
     //
 
-    // HVB (RWA009-A) legal update doc
-    string constant DOC = "QmQx3bMtjncka2jUsGwKu7ButuPJFn9yDEEvpg9xZ71ECh";
-
     function officeHours() public override returns (bool) {
-        return false;
+        return true;
     }
 
     function actions() public override {
+
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        // onboardNewCollaterals();
+        onboardNewCollaterals();
         // offboardCollaterals();
 
-        // ---------------------- CU DAI Vesting Streams -----------------------
-        // https://vote.makerdao.com/polling/QmQJ9hYq#poll-detail
-        // NOTE: ignore in goerli
-
-        // ---------------------- SPF Funding Transfers ------------------------
-        // https://forum.makerdao.com/t/mip55c3-sp6-legal-domain-work-on-greenlit-collateral-bibta-special-purpose-fund/17166
-        // https://vote.makerdao.com/polling/QmdaG8mo#vote-breakdown
-        // NOTE: ignore in goerli
+        // lock RWA007 Token in the URN
+        ERC20Like(RWA007).approve(RWA007_A_URN, 1 * WAD);
+        RwaUrnLike(RWA007_A_URN).lock(1 * WAD);
 
-        // ------------------- GRO-001 MKR Stream Clean-up ---------------------
-        // https://forum.makerdao.com/t/executive-inclusion-gro-001-mkr-vesting-stream-clean-up/17820
-        // NOTE: ignore in goerli
-
-        // -------------------- Update HVB Legal Documents ---------------------
-        // https://forum.makerdao.com/t/poll-inclusion-request-hvbank-legal-update/17547
-        // https://vote.makerdao.com/polling/QmX81EhP#vote-breakdown
-        bytes32 ilk                      = "RWA009-A";
-        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress(
-            "MIP21_LIQUIDATION_ORACLE"
-        );
-
-        ( , address pip, uint48 tau, ) = RwaLiquidationLike(
-            MIP21_LIQUIDATION_ORACLE
-        ).ilks(ilk);
-
-        require(pip != address(0), "Abort spell execution: pip must be set");
-
-        // Init the RwaLiquidationOracle to reset the doc
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            ilk,       // ilk to update
-            0,         // price ignored if init() has already been called
-            DOC,       // new legal document
-            tau        // old tau value
-        );
+        DssExecLib.setChangelogVersion("1.14.1");
     }
 }
 
 contract DssSpell is DssExec {
     constructor() DssExec(block.timestamp + 30 days, address(new DssSpellAction())) public {}
-}
+}
```

Comment:
We could leverage [`editorconfig`](https://editorconfig.org/) to prevent issues like that from happening.

Happy to create a PR with the config after the spell is merged.